examples/node_demo.js

```diff
@@ -1,7 +1,7 @@
-var Metrics = require('cd-metrics');
+var Metrics = require('../js/index.js');
```

Comment:
This line should be changed back so that the node_demo gets the cd-metrics node module from the node repository